test/e2e-tests.sh

```diff
@@ -41,6 +41,14 @@ chains_patch_spire
 
 failed=0
 
+namespace="tekton-chains"
+
+# Check if the NAMESPACE environment variable is set
+if [[ ! -z "$NAMESPACE" ]]; then
+  namespace="$NAMESPACE"
+fi
+echo "Using namespace: $namespace"
```

Comment:
I think this is ok, but it has the potential of being out of sync with the logic in the tests.